compiler/src/error.rs

```diff
@@ -56,7 +56,10 @@ impl fmt::Display for CompileError {
         }?;
 
         // Print line number:
-        write!(f, " at line {:?}", self.location.row())
+        match &self.error {
+            CompileErrorType::Parse(..) => Ok(()),
```

Comment:
Hmm. This exception for the parse error is not ideal, but we can improve on this later.

Reply:
Please implement the todo at this line: https://github.com/RustPython/RustPython/blob/ef1d5433df31a3c4ba7e58250b75de85ca76eaa1/compiler/src/error.rs#L17

Then we have proper locations on all compiler errors (where possible).

Reply:
Done